API/generate-api-tokens.md

```diff
@@ -0,0 +1,20 @@
+## How to generate API authentication tokens
+
+In order to generate API tokens to be used instead of Lime Field passwords in public scripts you need:
+
+1. Valid Lime Field account
```

Comment:
```suggestion
1. A valid Lime Field admin account
```

Reply:
Account does not have to be admin account. Changed to «A valid»